content/docs/how-we-work/coding-principles-standards.md

```diff
@@ -46,6 +46,30 @@ In our projects, we strictly adhere to the established coding standards. These s
 
   - When writing code, our code should solve ONLY and EXACTLY the use case that we are trying to support. In many instances, we have a desire to make things MORE FLEXIBLE in the FUTURE but that is code we will have to support without knowing whether any of those use cases will ever come up. Another big reason for wanting to be FLEXIBLE is that refactoring is hard, and it feels easier to have flexibility at the front, but that is why we write automated tests, so refactoring will not be as painful.
 
+## Quality as Teamwork
```

Comment:
This is unrelated. Maybe you need to rebase once?

Reply:
I did rebase this. I'll try it again. Thanks!